tools/metricsdocs/metricsdocs.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/monitoring"
+)
+
+// constant parts of the file
```

Comment:
I would suggest please to drop the comments here and on line 23
the code is self explanatory

Reply:
removed